client/me/concierge/controller.js

```diff
@@ -51,8 +51,15 @@ const reschedule = ( context, next ) => {
 	next();
 };
 
+const siteSelector = ( context, next ) => {
+	context.getSiteSelectionHeaderText = () =>
+		'Please select a business site to book a Concierge session';
```

Comment:
How about a slight tweak to:

```
<span>Please select a site for your <strong>Business Concierge Session</strong></span>
```

Reply:
Oh! This text should be translated too... So I guess it should be something like:

```js
context.getSiteSelectionHeaderText = () =>
	i18n.translate(
		'Please select a site for your {{strong}}Business Concierge Session{{/strong}}',
		{ components: { strong: <strong /> } }
	);```

Reply:
Great point, updated.